Cloud Application Manager/Tutorials/ebcli-tutorial.md

```diff
@@ -0,0 +1,442 @@
+
+
+## Deploy an application box
+
+Use the ebcli to deploy an application box in your Cloud Application Manager or your appliance account.
+
+**Terminal Command**
+
+```
+ebcli deploy [-w <WORKSPACE_ID>] [-t [<TAG> [<TAG> ...]]] [-c [<CLAIM> [<CLAIM> ...]]] input
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| -w | Workspace id where the box will be imported |
+| –t | Tag or Tags which will mark and isolate your application instances and bindings. i.e. 'production us-east-1' |
+| –c | Claims to select the correct deployment policies to be used by instances. i.e. 'ubuntu us-east-1' |
+| –input | Application's JSON document to deploy |
+
+
+### Instance lifecycle using ebcli
+
+## Get instances
+
+Use ebcli to list your instances.
+
+**Terminal Command**
+
+```
+ebcli instances [-f [<FIELD> [<FIELD> ...]]] [-t [<TAG> [<TAG> ...]]] [-w “<WORKSPACE ID>”] 
+```
+
+**Parameters**
+
+| Get Boxes Option | Description |
+|-------------------|-------------|
+| -f | Field or Fields you want to be listed. |
+| –t | Tag or Tags used to filter your instances. i.e. 'production us-east-1' |
+| –w | Workspace which contains the instances you want to list. |
+
+
+## Reconfigure an Instance
+
+Use ebcli to execute reconfigure operation on an instance which is powered on.
+
+**Terminal Command**
+
+```
+ebcli reconfigure ”<instance-id>”
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| instance-id | ID of the Instance which boxes you want to reconfigure. |
+
+
+## Reinstall an Instance
+
+Use ebcli to execute reinstall operation on an instance which is powered on.
+
+**Terminal Command**
+
+```
+ebcli reinstall ”<instance-id>”
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| instance-id | ID of Instance you want to power on. |
+
+
+## Terminate an Instance
+
+Use ebcli to execute terminate operation on an instance which is provisioned.
+
+***You can't connect to, restart or recover an instance after you've terminated it.***
+
+Before you terminate the instance, verify that you won't lose any data by checking that your volumes won't be deleted on termination and that you've copied any data that you need from your instance volumes to a persistent storage service.
+
+When an instance terminates, the data on any instance store and volumes associated with that instance are deleted.
+
+After you terminate an instance, it remains visible in the your cloud provider console for being deprovisioned short while, and then the entry is automatically deleted. After an instance is terminated, resources such as tags and volumes are gradually disassociated from the instance.
+
+When you terminate an instance, the whole history of the instance is kept in Cloud Application Manager and you can use it as a reference for other instances, to copy the variables or to clone the instance. The instance in your cloud provider is terminated or deprovisioned.
```

Comment:
Ok, I see that I was referring to above is here, but I will put it on top

Reply:
done